main.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+cmd_build() {
+  short_hash=$(git rev-parse --short HEAD)
+
+  echo "Building container"
+  docker build \
+      -t 864879987165.dkr.ecr.us-east-1.amazonaws.com/calm/localstripe:latest \
+      -t 864879987165.dkr.ecr.us-east-1.amazonaws.com/calm/localstripe:${short_hash} .
+}
+
+cmd_integ() {
+  docker build -t calm_localstripe:test .
```

Comment:
can you change `test` to `commit id` so that multiple builds will not interfere with each other.

Reply:
This is just a local build to run the tests against, it's not going to ECR